Jellyfin.Plugin.Webhook/Helpers/DataObjectHelpers.cs

```diff
@@ -159,6 +159,14 @@ public static Dictionary<string, object> AddBaseItemData(this Dictionary<string,
                     dataObject["AirTime"] = episode.Series.AirTime;
                 }
 
+                if (episode.Series?.ProviderIds is not null)
+                {
+                    foreach (var (providerKey, providerValue) in episode.Series.ProviderIds)
+                    {
+                        dataObject[$"SeriesProvider_{providerKey.ToLowerInvariant()}"] = providerValue;
+                    }
+                }
```

Comment:
```suggestion
                if (episode.Series?.ProviderIds is not null)
                {
                    var providers  = new Dictionary<string, object>();
                    dataObject["SeriesProviders"] = providers;
                    foreach (var (providerKey, providerValue) in episode.Series.ProviderIds)
                    {
                        providers  [providerKey.ToLowerInvariant()] = providerValue;
                    }
                }
```

why not make this a new object instead?

Reply:
why not using use ProviderIds directly ?
`dataObject[SeriesProviders] = episode.Series.ProviderIds`
or
`dataObject[SeriesProviders] = episode.Series.ProviderIds.ToDictionary(x => x.Key,  x => x.Value)`
if you want a different reference.

looking at the rest of the existing code. there seems to be a conscious effort to avoid objects (look at how media streams are assigned). I don't know what the reason is. changing those could be a breaking change for people.